crates/sqllib/src/variant.rs

```diff
@@ -426,7 +429,9 @@ impl Variant {
     #[doc(hidden)]
     pub fn index_string<I: AsRef<str>>(&self, index: I) -> Variant {
         match self {
-            Variant::Map(value) => match value.get(&Variant::String(index.as_ref().to_string())) {
+            Variant::Map(value) => match value.get(&Variant::String(SqlString::from(
+                index.as_ref().to_string(),
```

Comment:
This is a bit unfortunate. It should be possible to do map lookup by reference, but I'm not sure how to do this here.